apps/champions/lib/champions/users.ex

```diff
@@ -88,19 +83,37 @@ defmodule Champions.Users do
     Currencies.add_currency(user.id, Currencies.get_currency_by_name!("Gems").id, 500)
   end
 
+  defp add_campaigns_progress(user) do
+    campaigns = GameBackend.Campaigns.get_campaigns()
+
+    Enum.each(campaigns, fn campaign ->
+      if campaign.campaign_number == 1,
+        do:
+          GameBackend.Campaigns.insert_campaign_progress(%{
+            game_id: @game_id,
+            user_id: user.id,
+            campaign_id: campaign.id,
+            level_id: campaign.levels |> hd() |> Map.get(:id)
```

Comment:
This will be incorrect if the `campaign.levels` are not sorted right?  the sort was removed from `get_campaigns`

Reply:
Yes indeed! Fixed by sorting the list.